ActindoWorkFlow/UserInterface/03a_Processes.md

```diff
@@ -0,0 +1,634 @@
+
+    - **All**  
+    Select this option to display all actions, regardless of their status.
+
+    - **Active**  
+    Select this option to display the actions that are currently active.
+
+    - **Done**  
+    Select this option to display the actions that have been already executed.  
+
+    - **Error**  
+    Select this option to display the actions that have given an error.
+
+    - **Error; automatic retry**  
+    Select this option to display the actions that have given an error but are set for automatic retry.
+
+    > [Info] This setting depends on the action and is usually preconfigured.
+
+    - **Suspended**  
+    Select this option to display the actions that have been manually paused.
+
+    - **Process aborted**  
+    Select this option to display the actions included in a process that has been manually ended.
+
+
+
+- ![Search](/Assets/Icons/Search.png "[Search]") (Search)  
+    Click this button to display the search bar and search for a workflow.
+
+- ![Refresh](/Assets/Icons/Refresh01.png "[Refresh]") (Refresh)  
+    Click this button to update the list of workflows.
+
+- ![Columns](/Assets/Icons/Columns.png "[Columns]") Columns (x)  
+    Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The x indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](/Assets/Icons/Filter.png "[Filter]") Filter (x)  
+    Click this button to display the filter bar and customize the active filters. The x indicates the number of filters that are currently active.
+
+
+The list displays all actions. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+  - [x]    
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all actions in the list are selected.
+
+  Depending of the process status, the editing toolbar may display different options.
+
+  - ![Retry](/Assets/Icons/Retry01.png "[Retry]") (Retry)  
+    Click this button to retry the action. This button is displayed when the action status is *Error*.
+  - ![Suspend](/Assets/Icons/Pause.png "[Suspend]") (Suspend)  
+    Click this button to suspend the action temporarily. This button is displayed when the action status is *In Progress*. When clicking this button, the action status changes to *Suspended*.
+  - ![Start](/Assets/Icons/Play.png "[Start]") (Start)  
+    Click this button to resume the action. This button is displayed when the action status is *In Progress*. When clicking this button, the action status changes again to *In Progress*. The warning sign in the *Suspended Actions* column of the *Processes* view is no longer displayed.
+  - [REMOVE DEFER TIME]  
+    Click this button to eliminate the defer time preconfigured in the action, if any. The action is then executed immediately. This button is displayed when the action status is *In Progress*.
```

Comment:
verschieben in Zeile 262/263 (zwischen Filter button und "The list displays all actions...")